specs/apiCall.spec.js

```diff
@@ -46,23 +48,53 @@ describe('apiCall', () => {
 
       expect(test).to.deep.include({
         name: 'Checking login response',
-        expects: [
-          {
-            'userResponse.message': 'to.be User logged Successfully',
-          },
-        ],
+        expects: [{
+          'userResponse.message': 'to.be User logged Successfully',
+        }],
       })
     })
 
 
     it('Should evaluate global fake data', () => {
       const apiCallConfig = new ApiCall('./fixtures/fixture.spec.yaml', apiPort);
-      const apiCall = first(apiCallConfig.apiCalls)
+      const apiCall = first(apiCallConfig.apiCalls.swagger)
       const body = apiCall.data
       expect(body.email).to.be.a('string')
       expect(body.email).not.to.have.string('!faker')
 
       expect(body.password).not.to.be.a('string')
     })
+
+
+    it('Should repeat the file if the repeat tag is present change the it text', () => {
```

Comment:
BTW, [mocha explicitly discourages](https://mochajs.org/#arrow-functions) using arrow functions in unit tests because of the way `this` is bound (or not). Might need to change the lint rules.